shiny/ui/_include_helpers.py

```diff
@@ -0,0 +1,300 @@
+__all__ = ("include_javascript", "include_css", "include_html")
+
+import glob
+import os
+import shutil
+import tempfile
+from typing import Dict, Literal, Tuple
+
+# TODO: maybe these include_*() functions should actually live in htmltools?
+from htmltools import HTMLDependency, Tag, TagAttrArg, tags
+from htmltools._util import hash_deterministic
+
+from .._docstring import add_example
+
+# TODO: it's bummer that, when method="link_files" and path is in the same directory
+# as the app, the app's source will be included. Should we just not copy .py/.r files?
+
+
+@add_example()
+def include_javascript(
```

Comment:
I think `include_js` is a better name -- it's shorter, still clear, and has a similar form as `include_css`.